packages/ui/src/components/va-radio/VaRadio-validation.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <VbCard title="va-input validation">
+    <va-radio
```

Comment:
Let's have this demo in the `VaRadio.demo.vue`. Later, we will do a script to transform all of these demos to storybook, so it is better it the way other demos have.